schematic/ICs.dcm

```diff
@@ -12,6 +12,11 @@ K led driver buck 1a switching pwm analog
 F https://www.diodes.com/assets/Datasheets/AL8860.pdf
 $ENDCMP
 #
+$CMP DT1042-04SO-7
+D 4 Channel Low Capacitance TVS Diode Array, Typical 9V at 10A 100ns
+K ESD TVS Diode
+$ENDCMP
+#
```

Comment:
Here is where the conflict arises - I think I can just tell Git to include both ICs, but I don't want to break anything